frontend/src/assets/scss/form.scss

```diff
@@ -19,10 +19,13 @@
   color: inherit;
 }
 
-
 .el-input-group--append {
+  .el-input__wrapper {
+    border-radius: 4px 0 0 4px !important;
+  }
+
   .el-input-group__append {
-    @apply bg-white h-10 hover:bg-gray-50;
+    @apply bg-white h-10 w-10 hover:bg-gray-50;
```

Comment:
Let's put this `w-10` on a more specific CSS class
Otherwise, other input groups (which don't have squared buttons, like the search + filters) won't have the desired UI.